src/app/functions/server/mockL10n.ts

```diff
@@ -15,11 +15,15 @@ import type { LocaleData } from "./l10n";
 /* eslint-disable @typescript-eslint/no-var-requires */
 
 export function getEnL10nBundlesSync(): LocaleData[] {
+  // Ignored for test coverage, since tests don't run in a Webpack context:
+  /* c8 ignore next 2 */
```

Comment:
Should this be `next 2` or `next 3`, or maybe i'm confused by line counting on ternary operators.

Reply:
Same here; it's `2`, because the Node context branch does get hit in unit tests.